runtime/hydradx/src/governance/mod.rs

```diff
@@ -146,13 +146,8 @@ impl frame_support::traits::tokens::Pay for PayFromTreasuryAccount {
 
 impl pallet_treasury::Config for Runtime {
 	type Currency = Balances;
-	type ApproveOrigin = EitherOf<EnsureRoot<AccountId>, Treasurer>;
```

Comment:
where did this go?

Reply:
`ApproveOrigin` is used only in one extrinsic, `approve_proposal`. The extrinsic was deprecated and replaced by `spend` extrinsic and `SpendOrigin`.